src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -25,7 +25,65 @@
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Update="ProjectSystem\VS\PropertyPages\DebugPageControl.xaml.cs">
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
globbing is in 15.3 but not 15.2?  I'll change this.  Don't have permissions to push to this repo though